src/GSoft.Extensions.MediatR.Tests/GSoft.Extensions.MediatR.Tests.csproj

```diff
@@ -15,6 +15,7 @@
   <ItemGroup>
     <PackageReference Include="GSoft.Extensions.Xunit" Version="1.0.1" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.1" />
+    <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="17.4.1" Condition=" '$(OS)' != 'Windows_NT' " />
```

Comment:
Required to run tests on `net462` on Linux. `net462` validates the library on `netstandard2.0`.